app/setup.py

```diff
@@ -43,9 +43,10 @@
 }
 
 CSP_POLICY = {
-    'default-src': ["'self'", ],
-    'script-src': ["'self'", 'https://www.google-analytics.com', ],
-    'img-src': ["'self'", 'data:', 'https://www.google-analytics.com', 'https://cdn.ons.gov.uk']
+    'default-src': ["'self'", 'https://cdn.ons.gov.uk', ],
+    'font-src': ["'self'", 'data:', 'https://cdn.ons.gov.uk', ],
+    'script-src': ["'self'", 'https://www.google-analytics.com', 'https://cdn.ons.gov.uk', ],
+    'img-src': ["'self'", 'data:', 'https://www.google-analytics.com', 'https://cdn.ons.gov.uk', ]
 }
```

Comment:
We're also missing a `connect-src` policy here with `'https://www.google-analytics.com'`.

Reply:
Hi Neale, as discussed with Sam. The `connect-src` for `https://www.google-analytics.com` for be a separate change for an existing issue.

Reply:
@nealedj I certainly think you've found found a bug in the household module. The execution of this expression does need to be deferred but it appears the issue is related to the content security policy. This is an existing issue that already appears in census pages on preprod: https://preprod-surveys-launch.eq.ons.digital/